detox/local-cli/detox-test.js

```diff
@@ -45,6 +45,8 @@ program
     '[Android Only] Launch Emulator in headless mode. Useful when running on CI.')
   .option('-w, --workers <n>',
     '[iOS Only] Specifies number of workers the test runner should spawn, requires a test runner with parallel execution support (Detox CLI currently supports Jest)', 1)
+  .option('-n, --device-name [name]',
+    'Override the device name specified in a configuration. Useful for running a single build configuration on multiple devices.')
```

Comment:
Please update docs: https://github.com/wix/detox/blob/master/docs/APIRef.DetoxCLI.md

Reply:
Ah, I'd forgotten about that. Done, thanks.